pkg/bee/client.go

```diff
@@ -335,6 +335,22 @@ func (c *Client) SendPSSMessage(ctx context.Context, nodeAddress swarm.Address,
 	return c.api.PSS.SendMessage(ctx, nodeAddress, publicKey, topic, bytes.NewReader(data))
 }
 
+type SOC struct {
+	Reference string
+}
+
+// SendPSSMessage triggers a PSS message with a topic and recipient address
+func (c *Client) SubmitSOC(ctx context.Context, owner, ID, signature string, data []byte) (SOC, error) {
```

Comment:
maybe for sake of uniformity, call it `UploadSOC`? since other APIs are also `UploadChunk|File|Bytes` etc